gatsby-plugin-material-ui/src/__tests__/gatsby-browser.test.js

```diff
@@ -1,4 +1,15 @@
-import { onInitialClientRender } from "../gatsby-browser";
+import { onInitialClientRender } from '../gatsby-browser';
+// import SPMock from '../.cache/styles-provider-props';
```

Comment:
If it's for manually testing the test, I would add a comment explaining why it's here. Otherwise, I would remove the comment.